src/DisplayData/Station/StationDisplayData.php

```diff
@@ -120,7 +197,7 @@ private function getStationObsWithPictures(): array
 
     private function sortObsByDAte(Observation $obsA, Observation $obsB)
     {
-        return $obsB->getDateObs() <=> $obsA->getDateObs();
+        return $obsB->getObsDate() <=> $obsA->getObsDate();
```

Comment:
Just getDate of Observation is fine : $obs->getDate() is pretty simple to read and understand. 
No need for getDateObs or getObsDate